cpp_exercise/src/lib/filehandlerlib.cc

```diff
@@ -0,0 +1,176 @@
+#include "filehandlerlib.h"
+#include "ipcexceptionlib.h"
+
+/**
+ *
+ * @brief FileHandler Constructor for saving the file name
+ * 
+ * 
+ */
+
+FileHandler::FileHandler(const std::string & filename) : _file_name(filename){
+
+}
+
+/**
+ * @brief Destroy the File Handler:: File Handler object and close the file 
+ * 
+ */
+
+FileHandler::~FileHandler(){
+
+    close(_fd);
```

Comment:
If you used filestream instead of file descriptor, closing will be automatic.

Reply:
I am going to change it later in my refinement and finding errors story